cogs/sfu-utilities.py

```diff
@@ -79,9 +82,35 @@ def __init__(self, bot):
 
         await self.bot.send_file(ctx.message.channel, path)
 
+    @commands.command(name="report")
+    async def report(self):
+        """Show the SFU Campus Report"""
+        with urllib.request.urlopen(ROAD_API) as roadReport:
+            results = json.loads(roadReport.read().decode('utf-8'))
+        # await self.bot.say(results)
+        embed = discord.Embed()
+        embed.title = "SFU Campus Report"
+        vanAnnounce = results["campuses"]["vancouver"]["announcements"] or "No updates."
+        surreyAnnounce = results["campuses"]["surrey"]["announcements"] or "No updates."

```

Comment:
I already said to throw these up as constants....

Reply:
Fixed, thanks!